### PR TITLE
Add crc command

### PR DESCRIPTION
**Pull request by rjoleary**
Signed-off-by: Ryan O'Leary <ryanoleary@google.com>

### CODE REVIEW

**Comment by rjoleary** [comment]
It might make sense to merge this with cksum. I opted not to because cksum is a non-standard checksum trying to follow the GNU implementation closely.

File: cmds/crc/crc.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		f, err := os.Open(flag.Arg(0))
+		if err != nil {
+			log.Fatal(err)
+		}
+		defer f.Close()
+		r = f
+	default:
+		log.Fatal("expected 0 or 1 positional args")
+	}
+	data, err := ioutil.ReadAll(r)
```
Comment (rminnich) [inline cmds/crc/crc.go]: I think you should make this work in a streaming mode. The package supports this, see
https://golang.org/src/hash/crc32/crc32.go?s=6349:6401#L200

Comment (rjoleary) [reply cmds/crc/crc.go]: Done!

**Comment by rjoleary** [comment]
> as I read the docs it should be easy to make this streaming, I think that would be nice to have.

It should be streaming now. For example, when I run `./crc /dev/zero`, it runs forever without running out of memory.

**Pull request merged**